### PR TITLE
ELEC-449: Map Ids

### PR DESCRIPTION
**Pull request by ckitagawa**
This maps CAN message IDs and data to outputs on the driver display. This is very fragile and should be made more robust but in an effort to have something for Sunday this should suffice. Let me know if any of the used math/message don't make sense.

### CODE REVIEW

File: client/src/ts/can_msg_defs.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// For setting the CAN device
+export const enum CanDevice {
```
Comment (tituschow) [inline client/src/ts/can_msg_defs.ts]: I'm assuming this only exists as a hack?

File: client/src/ts/dd_main.ts [hunk 2]
```diff
@@ -178,52 +180,53 @@ const ws = new WebSocket('ws://localhost:8080/ws');
...
-      solarReadout.value(msg.data);
+  switch (can_id.msgId()) {
+    case canDefs.CanMessage.CAN_MESSAGE_SOLAR_DATA_REAR:  // Not working.
+      // solarReadout.value(msg.data);
       break;
-    case motorPowerLevel:
-      motor_power.addData({x: msg.timestamp, y: msg.data});
-      motorReadout.value(msg.data);
+    case canDefs.CanMessage.CAN_MESSAGE_MOTOR_VELOCITY:
+      speedDial.value((msg.data32[0] + msg.data32[1]) / 2 * 3.6);
```
Comment (tituschow) [inline client/src/ts/dd_main.ts]: Magic multiplier?

File: client/src/ts/can_msg_defs.ts (hunk 1 above)
Comment (ckitagawa) [reply client/src/ts/can_msg_defs.ts]: Yeah.

File: client/src/ts/dd_main.ts (hunk 2 above)
Comment (ckitagawa) [reply client/src/ts/dd_main.ts]: Done.

**Comment by ckitagawa** [comment]
This is now replaced with a better version.

**Pull request closed without merging**